### ISSUE TITLE
Serious accessibility issues affecting visually impaired screen reader users

### ISSUE THREAD

**Issue by amirsol81** [QuiteRSS/quiterss#1346]
I'm using QuiteRSS 0.19.4 along with NVDA 2020.2 -- an open-source screen reader for the visually impaired, and have encountered the following serious issues:
1. NVDA fails to read the contents of the Document window. If I'm not mistaken, the Document window tends to display the text of the currently selected article. But NVDA remains silent there. This is quite serious.
2. Once the focus moves away from the tree view, returning to it via NVDA is very difficult. That is, pressing Tab or Shift+Tab around the app doesn't properly announce the currently highlighted control. So we should press Tab or Shift+Tab and then randomly press Up/Down to force NVDA to move the focus to the RSS tree view.
3. First-letter navigation doesn't force in the tree view at all. This means that we can move to a folder or RSS feed in the tree view based on its first letter or letters.
4. QuiteRSS apparently has controls such as Mark as Read, but either these controls have no keyboard equivalents, or they are not detected by NVDA.
5. Is QuietRSS menu-driven? If so, pressing ALT doesn't bring up the menus. Neither it is detectable by pressing Tab or Shift+Tab.
In short, QuietRSS, despite its awesome features and advanced functionality, hasn't been designed with accessibility and visually impaired users in mind. I hope these issues are fixed in the near future so that we can recommend it to blind users.
Thanks.

**Comment by amirsol81** [comment]
@Funcy-dcm @arhohryakov @TotalCaesar659 and others - any chance of giving accessibility issues for the visually impaired a top priority? QuiteRSS is almost unusable for the visually impaired with Windows screen readers such as NVDA and JAWS given the issues described in my report. It is so unfortunate that the best Windows RSS reader suffers from such a severe case of inaccessibility. Thanks.
